components/search/Cargo.toml

```diff
@@ -5,6 +5,7 @@ edition = "2021"
 description = "Library for search service related components."
 readme = "README.md"
 license = "MPL-2.0"
+exclude = ["/ios"]
```

Comment:
I know you're just following what the other crates are doing here, but I'm wondering why this is needed.  We don't publish our crates to crates.io and if we did, I think it's as likely as not that we would want these directories.

This shouldn't impact this PR, I'm just thinking out loud.  Maybe we could do something in a follow-up.

Reply:
That's a good point! Yeah pretty much decided to stay inline with other crates. I think that's a good idea to simply remove that exclusion for all crates to reduce confusion in the future -- I'll maybe make a follow up bug for discussion on that!